tests/libslic3r/CMakeLists.txt

```diff
@@ -1,44 +1,44 @@
 get_filename_component(_TEST_NAME ${CMAKE_CURRENT_LIST_DIR} NAME)
 
-add_executable(${_TEST_NAME}_tests 
-	${_TEST_NAME}_tests.cpp
-	test_3mf.cpp
-	test_aabbindirect.cpp
-	test_clipper_offset.cpp
-	test_clipper_utils.cpp
-	test_config.cpp
-	test_elephant_foot_compensation.cpp
-	test_geometry.cpp
-	test_placeholder_parser.cpp
-	test_polygon.cpp
-	test_mutable_polygon.cpp
-	test_mutable_priority_queue.cpp
-	test_stl.cpp
-	test_meshboolean.cpp
-	# test_marchingsquares.cpp
-	test_timeutils.cpp
-	test_voronoi.cpp
+add_executable(${_TEST_NAME}_tests
+        ${_TEST_NAME}_tests.cpp
+        test_3mf.cpp
+        test_aabbindirect.cpp
+        test_clipper_offset.cpp
+        test_clipper_utils.cpp
+        test_config.cpp
+        test_elephant_foot_compensation.cpp
+        test_geometry.cpp
+        test_placeholder_parser.cpp
+        test_polygon.cpp
+        test_mutable_polygon.cpp
+        test_mutable_priority_queue.cpp
+        test_stl.cpp
+        test_meshboolean.cpp
+        test_marchingsquares.cpp
+        test_timeutils.cpp
+        test_voronoi.cpp
     test_optimizers.cpp
     # test_png_io.cpp
     test_indexed_triangle_set.cpp
     ../libnest2d/printer_parts.cpp
-	)
+        )
 
 if (TARGET OpenVDB::openvdb)
     target_sources(${_TEST_NAME}_tests PRIVATE test_hollowing.cpp)
 endif()
-    
+
 target_link_libraries(${_TEST_NAME}_tests test_common libslic3r)
 set_property(TARGET ${_TEST_NAME}_tests PROPERTY FOLDER "tests")
 
 if (WIN32)
-	if ("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
-	orcaslicer_copy_dlls(COPY_DLLS "Debug" "d" output_dlls_Debug)
-	elseif("${CMAKE_BUILD_TYPE}" STREQUAL "RelWithDebInfo")
-	orcaslicer_copy_dlls(COPY_DLLS "RelWithDebInfo" "" output_dlls_Release)
-	else()
-	orcaslicer_copy_dlls(COPY_DLLS "Release" "" output_dlls_Release)
-	endif()
+        if ("${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
```

Comment:
Please fix your editor to use 4-space indentation. If you look at the root CMakeLists.txt, it's obvious 4-space indentation is the intention.